modules/contact/src/ContactAction.C

```diff
@@ -231,13 +321,32 @@ ContactAction::getSmoothingEnum()
   return MooseEnum("edge_based nodal_normal_based", "");
 }
 
+MooseEnum
+ContactAction::getOrderEnum()
```

Comment:
why not just call `AddVariableAction::getNonlinearVariableOrders()` directly? That's what we do in all the other actions.

Reply:
`AddVariableAction::getNonlinearVariableOrders()` sets a default order/family which will be incorrect for tangential mortar constraints. Rather than have a default, I thought it might be better to set the default at runtime based on normal vs tangential and then let the user override it.

Reply:
I don't understand. This method literally just passes the return value `AddVariableAction::getNonlinearVariableOrders();` through.

Reply:
Sorry, I changed the history of the commit. At the moment, yes, it's just a pass-through. Let me get the variable order working first; then it should be clear how to handle this.